apertium-kaa.kaa.lexc

```diff
@@ -21898,7 +21904,7 @@ qandayınan:qandayınan N1 ;
 qandayına:qandayına N1 ;
 qandayınsha:qandayınsha N1 ;
 qandaylıq:qandaylıq N1 ;
-qanday:qanday N1 ;
+qanday:qanday ADV ;
```

Comment:
This is an interrogative determiner or similar.  It might have interrogative adverb uses too (could you give an example?).  But it's not a plain adverb.